docs/_data/fxn_registry.json

```diff
@@ -0,0 +1,2064 @@
+{
+    "_readme": [
+        "This file is automatically generated by our phpDocGen.php and is",
+        "a part of our auto-documentation process. If you see a mistake,",
+        "please edit the documentation in the respective function files."
+    ],
+    "methods": [
```

Comment:
That's great!

Seems like you have done the hardest part concerning the website doc content :)